src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/ViewDataInfo.cs

```diff
@@ -32,6 +34,18 @@ public ViewDataInfo(object container, PropertyInfo propertyInfo, Func<object> va
             _valueAccessor = valueAccessor;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ViewDataInfo"/> class with info about a
+        /// <see cref="ViewDataDictionary"/> lookup which is evaluated when <see cref="Value"/> is read. 
+        /// It uses <see cref="PropertyInfo.GetValue(object)"/> on <paramref name="propertyInfo"/> passing parameter <paramref name="container"/>.
+        /// </summary>
+        public ViewDataInfo(PropertyInfo propertyInfo, object container)
```

Comment:
Agree the IntelliSense is a bit nicer as you've done it. But consistent ordering helps consume the API even more -- devs still see the available overloads after typing the first argument. Let's go w/ consistency.